### PR TITLE
[Excel](Custom Functions) Changing filenames so they match rollback

### PR DESCRIPTION
**Pull request by mscharlock**
We updated everything for the new template that Adam had but now that we rolled back and are holding Adam's template until later in Feb, I'm updating docs to have the correct filenames with yo office

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[099d694](https://github.com/OfficeDev/office-js-docs-pr/pull/661/commits/099d6947cbc6f8df2326bb2157a2d55c7864cf83)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[099d694](https://github.com/OfficeDev/office-js-docs-pr/pull/661/commits/099d6947cbc6f8df2326bb2157a2d55c7864cf83)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

File: docs/excel/custom-functions-best-practices.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-ms.date: 11/29/2018
```
Comment (mscharlock) [inline docs/excel/custom-functions-best-practices.md]: Sorry, will get rid of this change

**Comment by o365devx** [comment]
OPS Build status updates of commit _[099d694](https://github.com/OfficeDev/office-js-docs-pr/pull/661/commits/099d6947cbc6f8df2326bb2157a2d55c7864cf83)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-filenamesrollback/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-filenamesrollback/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201901032019495504-661%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9w68vZH53J%2BAsh5gsH41W4g0k592AEGsPakjY4RpxzE%3D&st=2019-01-03T20%3A16%3A04Z&se=2019-02-03T20%3A21%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-ms.date: 12/21/2018
+ms.date: 01/03/2018
```
Comment (kbrandl) [inline docs/excel/custom-functions-overview.md]: year should be 2019

**Comment by mscharlock** [comment]
Doesn't include changes to the tutorial because I'll do a mass update to that separately

File: docs/excel/custom-functions-overview.md [hunk 3]
```diff
@@ -31,16 +31,16 @@ If you use the [Yo Office generator](https://github.com/OfficeDev/generator-offi
 
 | File | File format | Description |
 |------|-------------|-------------|
-| **./src/functions/functions.js**<br/>or<br/>**./src/functions/functions.ts** | JavaScript<br/>or<br/>TypeScript | Contains the code that defines custom functions. |
-| **./src/functions/functions.json** | JSON | Contains metadata that describes custom functions and enables Excel to register the custom functions and make them available to end users. |
-| **./src/functions/functions.html** | HTML | Provides a &lt;script&gt; reference to the JavaScript file that defines custom functions. |
+| **./src/functions.js**<br/>or<br/>**./src/functions.ts** | JavaScript<br/>or<br/>TypeScript | Contains the code that defines custom functions. |
```
Comment (kbrandl) [inline docs/excel/custom-functions-overview.md]: files *names* are incorrect (should be **customfunctions.js** and **customfunctions.ts**)

File: docs/excel/custom-functions-overview.md [hunk 4]
```diff
@@ -31,16 +31,16 @@ If you use the [Yo Office generator](https://github.com/OfficeDev/generator-offi
...
+| **./config/functions.json** | JSON | Contains metadata that describes custom functions and enables Excel to register the custom functions and make them available to end users. |
```
Comment (kbrandl) [inline docs/excel/custom-functions-overview.md]: files *name* is incorrect (should be **customfunctions.json**)

File: docs/excel/custom-functions-overview.md [hunk 5]
```diff
@@ -31,16 +31,16 @@ If you use the [Yo Office generator](https://github.com/OfficeDev/generator-offi
...
+| **./index.html** | HTML | Provides a &lt;script&gt; reference to the JavaScript file that defines custom functions. |
 | **./manifest.xml** | XML | Specifies the namespace for all custom functions within the add-in and the location of the JavaScript, JSON, and HTML files that are listed previously in this table. |
 
 The following sections provide more information about these files.
 
 ### Script file
 
-The script file (**./src/functions/functions.js** or **./src/functions/functions.ts** in the project that the Yo Office generator creates) contains the code that defines custom functions and maps the names of the custom functions to objects in the [JSON metadata file](#json-metadata-file). 
+The script file (**./src/functions.js** or **./src/functions.ts** in the project that the Yo Office generator creates) contains the code that defines custom functions and maps the names of the custom functions to objects in the [JSON metadata file](#json-metadata-file).
```
Comment (kbrandl) [inline docs/excel/custom-functions-overview.md]: files *names* are incorrect (should be **customfunctions.js** and **customfunctions.ts**)

File: docs/excel/custom-functions-overview.md [hunk 6]
```diff
@@ -68,7 +68,7 @@ CustomFunctionMappings.INCREMENT = increment;
 
 ### JSON metadata file 
 
-The custom functions metadata file (**./src/functions/functions.json** in the project that the Yo Office generator creates) provides the information that Excel requires to register custom functions and make them available to end users. Custom functions are registered when a user runs an add-in for the first time. After that, they are available to that same user in all workbooks (i.e., not only in the workbook where the add-in initially ran.)
+The custom functions metadata file (**./config/functions.json** in the project that the Yo Office generator creates) provides the information that Excel requires to register custom functions and make them available to end users. Custom functions are registered when a user runs an add-in for the first time. After that, they are available to that same user in all workbooks (i.e., not only in the workbook where the add-in initially ran.)
```
Comment (kbrandl) [inline docs/excel/custom-functions-overview.md]: files *name* is incorrect (should be **customfunctions.json**)

**Pull request merged**